runtime/nls/j9vm/j9vm.nls

```diff
@@ -1932,3 +1932,10 @@ J9NLS_VM_CLASS_LOADING_ERROR_NON_INTERFACE.explanation=The specified interface i
 J9NLS_VM_CLASS_LOADING_ERROR_NON_INTERFACE.system_action=The JVM will throw an IncompatibleClassChangeError.
 J9NLS_VM_CLASS_LOADING_ERROR_NON_INTERFACE.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_VM_XVERIFYNONE_DEPRECATED=Options -Xverify:none and -noverify were deprecated in version 13.0 and will likely be removed in a future release.
```

Comment:
Does OpenJ9 support the -noverify option? Either it shouldn't be mentioned, or (+1) we should add a mapping and support the option for compatibility.

Reply:
I guess the java launcher converts -noverify to -Xverify:none